style.css

```diff
@@ -454,6 +454,22 @@ main h2, main h3, main h4, main h5, main h6 {
     margin-left: 2ch; /* extra indentation */
 }
 
+.decl.sorried {
+    background-color: #fff0f0;
+}
+
+.decl .sorry_msg {
+    background-color: #ffdada;
+    /* this pushes it flush with the colored margin */
+    margin-left: -8px;
```

Comment:
Do you have this deployed somewhere so that I can look at it?

Reply:
Working on it, CI seems to be swallowing my error messages. Thanks for the speedy review.